src/igvfd/schemas/changelogs/human_donor.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`human_donor.json`*
 
+### Schema version 4
+
+* Remove `external_resource` from base class `donor`.
```

Comment:
please drop the "from base class 'donor'. it is extra detail that is not very helpful in reading the changelog for human-donor.

Reply:
done